content/guides/debugging/_index.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "Incident Debugging Workflow"
+date: 2018-11-26T22:22:31-07:00
+draft: false
+aliases: [/integrations/workflow]
+weight: 1
+---
+
```

Comment:
Please perhaps add the sections index at the top as we do on all the other pages (quite handy for navigating before-hand)

```
- [Introduction](#introduction)
- [Workflow](#workflow)
- [Instrument your application](#instrument-your-application)
- [Setup Backends for Monitoring Metrics and Trace Data](#setup-backends-for-monitoring-metrics-and-trace-data)
- [Respond to alerts](#respond-to-alerts)
```
etc for all of them

Reply:
Done. Please see changes in new commit.